### PR TITLE
docs for v1.5.0 release

### PR DESCRIPTION
**Pull request by ruojunm**
## Description

Add release docs for the upcoming v1.5.0

## Rationale

NA

### CODE REVIEW

File: docs/release-notes/features.md [hunk 1]
```diff
@@ -32,10 +32,13 @@ the key features and their development status on Greenfield. Please stay tuned.
...
 |           Migrate bucket           | Greenfield supports users to migrate their bucket from one SP to another                                                                                                                                                                                                                                                                                                                                                                   | Testnet |
 |  Expiration time for group member  | Greenfield supports setting expired time for group members                                                                                                                                                                                                                                                                                                                                                                                 | Mainnet |
```
Comment (zjubfd) [inline docs/release-notes/features.md:34]: I think we have some feature enabled on mainnet few days ago, can we change the status of them as well

**Review by zjubfd — APPROVED** [review]

File: docs/release-notes/roadmap.md [hunk 2]
```diff
@@ -8,6 +8,11 @@ order: 2
 
 ## Upcoming
 
+### Pawnee Upgrade
```
Comment (KeefeL) [inline docs/release-notes/roadmap.md:11]: Duplicated, Please check in line21, I think we should introduce the next hardfork for v1.6 here.

**Review by zjubfd — APPROVED** [review]

File: docs/release-notes/roadmap.md [hunk 3]
```diff
@@ -8,24 +8,20 @@ order: 2
...
-The Ural hardfork introduces one significant feature in this upgrade.
-1. Support bucket migration: With this new feature, users can migrate their bucket to another storage provider(SP) as
-   the primary SP if they are not satisfied with the current service provided.
-
 ### Pawnee Upgrade
 The Pawnee hardfork will introduce several enhance features in this upgrade.
 1. Support atomic object update: To update an object currently, the process involves deleting the existing object on the
    chain and then creating a new object with the same name. This feature enables users to directly modify the content of
-   objects. For further information, please refer to [BEP346](https://github.com/bnb-chain/BEPs/pull/346).
+   objects. For further information, please refer to [BEP366](https://github.com/bnb-chain/BEPs/pull/346).
```
Comment (zjubfd) [inline docs/release-notes/roadmap.md:15]: I think this BEP is merged, so we can use the official link now.

Comment (ruojunm) [reply docs/release-notes/roadmap.md:15]: Sure .Fixed

**Review by zjubfd — APPROVED** [review]

**Review by KeefeL — APPROVED** [review]

**Pull request merged**